src/sentry/models/sentryapp.py

```diff
@@ -111,24 +110,6 @@ def visible_for_user(cls, request):
 
         return cls.objects.filter(status=SentryAppStatus.PUBLISHED)
 
-    @property
-    def organizations(self):
```

Comment:
Do we not use this anywhere else? Same thing for `teams`

Reply:
@MeredithAnya I couldn't find any other uses